### PR TITLE
Add layout and intro components

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import cn from "classnames"
+import s from "./Footer.module.css"
+import Container from "../Container/Container"
+
+interface IFooterProps {
+  className?: string
+}
+
+function Footer({ className }: IFooterProps) {
```
Comment (therealemjy) [inline src/components/Footer/Footer.tsx:10]: Just an FYI; React provides a type for functional components:
```typescript
const Footer: React.FC<IFooterProps> = ({ className }) => {
  ...
};
```

One of the advantages of using it is that your return value gets also typed.

**Review by therealemjy — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -10,16 +10,18 @@
...
     "react-dom": "^17.0.2",
     "react-scripts": "5.0.0",
     "typescript": "^4.6.2",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "react-app-rewired start",
```
Comment (coreyar) [inline package.json:22]: fyi: This project is "lightly maintained" - `craco` is probably a better option going forward

**Review by coreyar — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (baomastr) [reply package.json:22]: the reason of choosing app-rewired was the package react-app-rewire-postcss for postcss out of the box without configuration.
I think, on current stage it's good for speed up

File: src/components/Footer/Footer.tsx (hunk 1 above)
Comment (baomastr) [reply src/components/Footer/Footer.tsx:10]: yeah. that's nice option. shall use it

**Pull request merged**